src/main/drivers/accgyro/accgyro_spi_bmi160.c

```diff
@@ -254,12 +254,46 @@ static int32_t BMI160_do_foc(const extDevice_t *dev)
 
 extiCallbackRec_t bmi160IntCallbackRec;
 
-#if defined(USE_MPU_DATA_READY_SIGNAL)
+#ifdef USE_GYRO_EXTI
+// Called in ISR context
+// Gyro read has just completed
+busStatus_e bmi160Intcallback(uint32_t arg)
+{
+    gyroDev_t *gyro = (gyroDev_t *)arg;
+    int32_t gyroDmaDuration = cmpTimeCycles(getCycleCounter(), gyro->gyroLastEXTI);
+
+    if (gyroDmaDuration > gyro->gyroDmaMaxDuration) {
+        gyro->gyroDmaMaxDuration = gyroDmaDuration;
+    }
+
+    gyro->dataReady = true;
+
+    return BUS_READY;
+}
+
+void bmi160ExtiHandler(extiCallbackRec_t *cb)
+{
+    gyroDev_t *gyro = container_of(cb, gyroDev_t, exti);
+    // Ideally we'd use a time to capture such information, but unfortunately the port used for EXTI interrupt does
```

Comment:
Stupid question: why `ideally we'd use a timer to capture such information`?
PS: mistype `timer` as `time.`

Reply:
I've fixed the time/timer typo in `accgyro_mpu.c`, `accgyro_spi_bmi160.c` and `accgyro_spi_bmi270.c`.

The reason for this comment is that we're timing here using software to read the cycle counter. Were there to be a timer associated with this pin we'd be able to perform the timing in hardware which would be more consistent/accurate. Potentially an improvement for the future.